core/api/src/domain/users/phone-metadata-authorizer.ts

```diff
@@ -14,9 +14,9 @@ export const PhoneMetadataAuthorizer = ({
   const authorize = (
     phoneMetadata: PhoneMetadata | undefined,
```

Comment:
```suggestion
    phoneMetadata: PhoneMetadata | undefined | null,
```
is this the actual type we should have?